sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/SharedGalleryImageTests.java

```diff
@@ -35,18 +35,19 @@
 
 public class SharedGalleryImageTests extends ComputeManagementTest {
     private String rgName = "";
-    private final Region region = Region.US_WEST_CENTRAL;
-    private final String vmName = "javavm";
+    private final Region region = Region.US_WEST3;
+    private String vmName = "";
 
     @Override
     protected void initializeClients(HttpPipeline httpPipeline, AzureProfile profile) {
         rgName = generateRandomResourceName("javacsmrg", 15);
+        vmName = generateRandomResourceName("javavm", 12);
         super.initializeClients(httpPipeline, profile);
     }
 
     @Override
     protected void cleanUpResources() {
-        resourceManager.resourceGroups().beginDeleteByName(rgName);
+        resourceManager.resourceGroups().deleteByName(rgName);
```

Comment:
why need to wait for delete completion? this may take longer for the test to finish

Reply:
Fixed in the new version.